src/test/kotlin/com/terraformation/backend/documentproducer/api/DocumentsControllerTest.kt

```diff
@@ -188,15 +194,15 @@ class DocumentsControllerTest : ControllerIntegrationTest() {
     fun `updates a document by id`() {
       val otherMethodologyId = insertMethodology()
       val otherVariableManifestId = insertVariableManifest(methodologyId = otherMethodologyId)
-      val otherUserId = insertUser()
+      val otherUserId = insertUser(UserId(101))
```

Comment:
FYI, you can do `insertUser(101)` here for brevity (that's why the insert methods use `Any` as the types for their ID arguments).

Reply:
This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.

Reply:
This will be addressed in another PR into the feature branch, for now it is captured as a TODO in the main feature branch PR description.